source/database-connections/save-mongodb-connection.txt

```diff
@@ -0,0 +1,76 @@
+
+.. _rm-save-mongodb-connection:
+
+============================================
+Save a MongoDB Connection
+============================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+To start a migration job, Relational Migrator must connect to your MongoDB
+database using an account with :authrole:`readWrite` access. You can save a new
+MongoDB Connection:
+
+- When creating or modifying a :ref:`data migration job <rm-create-jobs>`
+- From the Database Connections bar in a project
+- From the Relational Migrator :guilabel:`Connections` page
+
+Before you Begin
+----------------
+
+- To save a connection to a MongoDB Atlas cluster, you must be :ref:`logged in <rm-atlas-log-in>` with Atlas.
+
+- For both Atlas and on-premises deployments, we recommend creating a separate
+  :ref:`MongoDB user <rm-mongodb-service-user>` for Relational Migrator.
+
+Steps
+-----
+
+To save a new connection from the :guilabel:`Connections` page:
+
+.. procedure::
+
+   .. step:: Add a new connection
+      
+      In the list of :guilabel:`MongoDB` connections, click :guilabel:`+ Add connection`.
+
+   .. step:: Enter the MongoDB connection string
+
+      a. In the :guilabel:`MongoDB connection string (URI)` field, input
+         your :manual:`MongoDB URI <mongodb-uri>`.
+
+      #. If it isn't included in the connection string, enter the
+         :guilabel:`Database` to connect to.
+      
+      #. If they aren't included in the connection string, enter the 
+         :guilabel:`Username` and :guilabel:`Password` of your
+         :ref:`Relational Migrator MongoDB user <rm-mongodb-service-user>`.
+         
+         Checking :guilabel:`Save password` saves the password securely
+         on your machine, so you don't have to enter the :guilabel:`Username`
+         and :guilabel:`Password` again when using the saved connection.
+      
+      If you leave the form :guilabel:`Database`, :guilabel:`Username`, 
+      or :guilabel:`Password` fields blank, Relational Migrator uses the
+      values from the URI.
+
+   .. step:: Specify a :guilabel:`Connection name` and optional :guilabel:`Environment tag`
```

Comment:
Updated this to use "Identify your connection", but for the include version of the procedure I left the action as the main step, since those are intended to be slightly streamlined as part of longer procedures. Thoughts? Also changed "specify" to "enter", per your point on jargon earlier.

Reply:
Thanks Nick! Appreciate this!